owl/SOMA-OBJ.owl

```diff
@@ -971,6 +1021,23 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#NetForce -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#NetForce">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAttribute"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasForceValue"/>
```

Comment:
hasForceValue has xsd:double as range. This covers the force magnitude, but what about its direction?

Reply:
Well one could compute from the position change, I assume.

Reply:
Yeah. Net forces in simulators are represented as vectors even, and this representation takes care of both magnitude and direction obviously.

Reply:
Would then using one of the vector types for the net force value help?

Reply:
ok, done